.changes/next-release/enhancement-cloudfront-36119.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "enhancement",
+  "category": "cloudfront",
+  "description": "Replace cryptographic functions from cryptography with awscrt for the sign command."
```

Comment:
nit: I think we usually do double-backticks on the service and command names.

Reply:
Fixed, thanks!